spec/ruby/core/time/succ_spec.rb

```diff
@@ -2,31 +2,33 @@
 require_relative 'fixtures/classes'
 
 describe "Time#succ" do
-  it "returns a new time one second later than time" do
-    suppress_warning {
-      @result = Time.at(100).succ
-    }
+  ruby_version_is ""..."3.0" do
```

Comment:
I would move this outside just above `describe "Time#succ" do`, so it's clear there is no Time#succ at all in Ruby 3.